tests/Unit/FilesystemTestCase.php

```diff
@@ -7,43 +7,71 @@
 
 abstract class FilesystemTestCase extends VirtualFilesystemTestCase {
 	protected $wprocket_structure = [
-		'example.org'                             => [
-			'index.html'      => '',
-			'index.html_gzip' => '',
-			'about'           => [
-				'index.html'             => '',
-				'index.html_gzip'        => '',
-				'index-mobile.html'      => '',
-				'index-mobile.html_gzip' => '',
-			],
-			'category'        => [
-				'wordpress' => [
+		'wp-rocket' => [
+			'example.org'                             => [
+				'index.html'      => '',
+				'index.html_gzip' => '',
+				'about'           => [
+					'index.html'             => '',
+					'index.html_gzip'        => '',
+					'index-mobile.html'      => '',
+					'index-mobile.html_gzip' => '',
+				],
+				'category'        => [
+					'wordpress' => [
+						'index.html'      => '',
+						'index.html_gzip' => '',
+					],
+				],
+				'blog'            => [
+					'index.html'      => '',
+					'index.html_gzip' => '',
+				],
+				'en'              => [
 					'index.html'      => '',
 					'index.html_gzip' => '',
 				],
 			],
-			'blog'            => [
-				'index.html'      => '',
-				'index.html_gzip' => '',
-			],
-			'en'              => [
+			'example.org-Greg-594d03f6ae698691165999' => [
 				'index.html'      => '',
 				'index.html_gzip' => '',
 			],
 		],
-		'example.org-Greg-594d03f6ae698691165999' => [
-			'index.html'      => '',
-			'index.html_gzip' => '',
+		'min' => [
+			'1' => [
+				'5c795b0e3a1884eec34a989485f863ff.js'     => '',
+				'5c795b0e3a1884eec34a989485f863ff.js.gz'  => '',
+				'fa2965d41f1515951de523cecb81f85e.css'    => '',
+				'fa2965d41f1515951de523cecb81f85e.css.gz' => '',
+			],
 		],
+		'busting' => [
+			'1' => [
+				'wp-content' => [
+					'themes' => [
+						'storefront' => [
+							'assets' => [
+								'js' => [
+									'navigation.min-2.5.3.js'    => '',
+									'navigation.min-2.5.3.js.gz' => '',
+								],
+							],
+							'style-2.5.3.css'    => '',
+							'style-2.5.3.css.gz' => '',
+						],
+					],
+				],
+			]
+		]
 	];
 	protected $cache_path;
 
 	public function setUp() {
-		$this->structure['wp-rocket'] = $this->wprocket_structure;
+		$this->structure['cache'] = $this->wprocket_structure;
```

Comment:
@Tabrisrp Delete this line of code. It's not needed anymore. The parent `setUp()` method will take care of instantiating the filesystem for you. [See it here](https://github.com/wp-media/phpunit/blob/master/Unit/VirtualFilesystemTestCase.php#L53).